### PR TITLE
 #900 Add it tests for streamApp

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #900)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by saivirtue** [comment]
retry it

**Comment by saivirtue** [comment]
retry confiurator

**Comment by saivirtue** [comment]
retry configurator

**Comment by saivirtue** [comment]
retry it

File: ohara-it/src/test/scala/com/island/ohara/it/agent/k8s/TestK8SStreamApp.scala [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+import org.scalatest.Matchers
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration.{FiniteDuration, _}
+import scala.concurrent.{Await, Future}
+class TestK8SStreamApp extends IntegrationTest with Matchers {
+
+  private[this] var bkName: String = _
+  private[this] var wkName: String = _
+  private[this] val instances = 1
```
Comment (chia7712) [inline ohara-it/src/test/scala/com/island/ohara/it/agent/k8s/TestK8SStreamApp.scala]: Do you plan to test multi-instances for streamapp?

**Comment by chia7712** [comment]
It would be better to check the **behavior** of a running streamapp. For example, you can submit a streamapp writing a file to (container's) local file, and then you retrieve the file by docker client and then check file content.

The other IT tests (such as broker and worker) have similar behavior which check whether container instance work well.

File: ohara-it/src/test/scala/com/island/ohara/it/agent/k8s/TestK8SStreamApp.scala (hunk 1 above)
Comment (saivirtue) [reply ohara-it/src/test/scala/com/island/ohara/it/agent/k8s/TestK8SStreamApp.scala]: sure. add a data test as well

**Comment by chia7712** [comment]
seems the jmx port is not available...Let me dig in it.

**Comment by chia7712** [comment]
retry it

**Comment by chia7712** [comment]
The failed test is traced by #936. I'm about to merge this PR.

**Pull request merged**